build/DockerfileTest

```diff
@@ -1,7 +1,4 @@
-FROM centos:7
-
-RUN adduser istio-workspace
-USER istio-workspace
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
Are we ok with moving target as a base image?

Reply:
Well.. Not really, but it's technically not any more moving than the one we had, as it's version locked by name, not tag. centos:7 vs uib8:latest would be same same was my thinking.

Reply:
Can we lock it to latest `8.0-131` then?

Reply:
8.0?

Reply:
yeah `8.0` would do, even though == latest :)

Reply:
```suggestion
FROM registry.access.redhat.com/ubi8/ubi-minimal:8.0
```